### ISSUE TITLE
Tippy x-placement always set to 'top' - how do I change this?

### ISSUE THREAD

**Issue by stefanReeves** [johndatserakis/vue-navigation-bar#2]
Hi @johndatserakis - love the look of the navbar, nice work. The navbar will fit a project I'm working on perfectly, but when using your example implementation, the placement of the popups is always set to 'top'.

Obviously, this is not what you want from a dropdown navbar. Any idea why this would be happening?

Thanks for any help you can give... here's a screenshot:
<img width="1006" alt="Screenshot 2019-06-04 at 17 27 07" src="https://user-images.githubusercontent.com/19836833/58892139-007c5f00-86ee-11e9-96c4-078271f3ae79.png">

**Comment by johndatserakis** [comment]
Hey there - sorry you're having trouble. 

Hmm, yea this is strange - I'm not seeing the same behavior in the demo project - I have a feeling it may have to do with whatever you're wrapping the component with. The wrapping div may give tippy the impression there isn't any room below the navbar. I think the setting of top is being set automatically from tippy - it's not something I set in the code I don't think.

Can you reproduce the situation in a codepen? Here's one I just built using the latest updates - https://codepen.io/johndatserakis/pen/ZNZYzo?editors=1011. Maybe you can adjust it so it's more like your project and we'll take a look.

**Comment by stefanReeves** [comment]
Thanks for the quick reply. I also assumed it must be another element causing tippy to place at the top. I stripped away anything that might interfere with tippy, but the problem persisted.

My project requires a lot of content to be included in the popups, so I decided to use vue-tippy instead, as I can then build the exact menu I need, instead of butchering yours to fit my client's crazy requirements ;)

Thanks again, as your plugin has led me to the correct solution.
